"./gradlew clean run" fails with exception due to changeset 47 and 52

```
[main] ERROR io.sdkman.changelogs.package$$anon$1 - URL https://download.java.net/java/GA/jdk11/28/GPL/openjdk-11+28_linux-x64_bin.tar.gz responded with text/html content-type detected
[main] INFO com.github.mongobee.Mongobee - Mongobee is releasing process lock.
Exception in thread "main" com.github.mongobee.exception.MongobeeException: Invalid url: https://download.java.net/java/GA/jdk11/28/GPL/openjdk-11+28_linux-x64_bin.tar.gz
        at com.github.mongobee.Mongobee.executeMigration(Mongobee.java:200)
        at com.github.mongobee.Mongobee.execute(Mongobee.java:156)
        at io.sdkman.DbMigration$.delayedEndpoint$io$sdkman$DbMigration$1(DbMigration.scala:10)
        at io.sdkman.DbMigration$delayedInit$body.apply(DbMigration.scala:5)
        at scala.Function0.apply$mcV$sp(Function0.scala:34)
        at scala.Function0.apply$mcV$sp$(Function0.scala:34)
        at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:12)
        at scala.App.$anonfun$main$1$adapted(App.scala:76)
        at scala.collection.immutable.List.foreach(List.scala:389)
        at scala.App.main(App.scala:76)
        at scala.App.main$(App.scala:74)
        at io.sdkman.DbMigration$.main(DbMigration.scala:5)
        at io.sdkman.DbMigration.main(DbMigration.scala)
Caused by: java.lang.reflect.InvocationTargetException
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at com.github.mongobee.Mongobee.executeChangeSetMethod(Mongobee.java:235)
        at com.github.mongobee.Mongobee.executeMigration(Mongobee.java:181)
        ... 12 more
Caused by: com.github.mongobee.exception.MongobeeChangeSetException: Invalid url: https://download.java.net/java/GA/jdk11/28/GPL/openjdk-11+28_linux-x64_bin.tar.gz
        at io.sdkman.changelogs.package$$anon$1.checkResourceAvailable(package.scala:45)
        at io.sdkman.changelogs.package$$anon$1.validUrl(package.scala:36)
        at io.sdkman.changelogs.package$$anon$1.validUrl(package.scala:26)
        at io.sdkman.changelogs.package$$anon$4.$anonfun$validUrl$1(package.scala:52)
        at io.sdkman.changelogs.package$$anon$4.$anonfun$validUrl$1$adapted(package.scala:52)
        at scala.collection.immutable.List.foreach(List.scala:389)
        at io.sdkman.changelogs.package$$anon$4.validUrl(package.scala:52)
        at io.sdkman.changelogs.package$$anon$4.validUrl(package.scala:49)
        at io.sdkman.changelogs.package$ValidationOps.validate(package.scala:84)
        at io.sdkman.changelogs.JavaMigrations.migrate047(JavaMigrations.scala:107)
        ... 18 more


FAILURE: Build failed with an exception.
```

--- a/src/main/scala/io/sdkman/changelogs/JavaMigrations.scala
+++ b/src/main/scala/io/sdkman/changelogs/JavaMigrations.scala
@@ -98,17 +98,6 @@ class JavaMigrations {
       .insert()
   }
 
-  @ChangeSet(order = "047", id = "047-add_openjdk_java_11.0.0", author = "mdeinum")
-  def migrate047(implicit db: MongoDatabase) = {
-    List(
-      Version("java", "11.0.0-open", "https://download.java.net/java/GA/jdk11/28/GPL/openjdk-11+28_linux-x64_bin.tar.gz", Linux64),
-      Version("java", "11.0.0-open", "https://download.java.net/java/GA/jdk11/28/GPL/openjdk-11+28_osx-x64_bin.tar.gz", MacOSX),
-      Version("java", "11.0.0-open", "https://download.java.net/java/GA/jdk11/28/GPL/openjdk-11+28_windows-x64_bin.zip", Windows))
-      .validate()
-      .insert()
-    Seq(Linux64, MacOSX, Windows).foreach(removeVersion("java", "11.ea.28-open", _))
-  }
-
   @ChangeSet(order = "048", id = "048-set_openjdk_java_11.0.0_default", author = "marc0der")
   def migrate048(implicit db: MongoDatabase) = setCandidateDefault("java", "11.0.0-open")
 
@@ -128,17 +117,6 @@ class JavaMigrations {
         platform = MacOSX))
       .validate()
       .insert()
-  }
-
-  @ChangeSet(order = "052", id = "052-add_openjdk_java_11.0.1", author = "mdeinum")
-  def migrate052(implicit db: MongoDatabase) = {
-    List(
-      Version("java", "11.0.1-open", "https://download.java.net/java/GA/jdk11/13/GPL/openjdk-11.0.1_linux-x64_bin.tar.gz", Linux64),
-      Version("java", "11.0.1-open", "https://download.java.net/java/GA/jdk11/13/GPL/openjdk-11.0.1_osx-x64_bin.tar.gz", MacOSX),
-      Version("java", "11.0.1-open", "https://download.java.net/java/GA/jdk11/13/GPL/openjdk-11.0.1_windows-x64_bin.zip", Windows))
-      .validate()
-      .insert()
-    Seq(Linux64, MacOSX, Windows).foreach(removeVersion("java", "11.0.0-open", _))
   }
 
   @ChangeSet(order = "054", id = "054-set_openjdk_java_11.0.1_default", author = "marc0der")

